website/blog/2021-11-14-Presto-Cluster-Memory-Tuning.md

```diff
@@ -0,0 +1,70 @@
+---
+author: George Wang
+authorURL: https://www.linkedin.com/in/george-wang-9a5a46190/
+title: Presto Cluster Memory Tuning
+---
+
+##Presto Memory Management
+
+Presto's memory management system is carefully designed to achieve high performing query engine in a distributed system. It defines the memory into two categories: user memory and system memory.
```

Comment:
there's also  a third category of "revocable memory", which is memory that can be spilled to disk if requested